migration/idempotent/001-base.sql

```diff
@@ -1576,6 +1602,27 @@ BEGIN
                 timescaledb.compress_orderby = 'time, value'
             ); $$, metric_table_name);
 
+        SELECT c.schema_name, c.table_name 
+        INTO STRICT _compressed_schema, _compressed_hypertable
+        FROM _timescaledb_catalog.hypertable h 
+        INNER JOIN _timescaledb_catalog.hypertable c ON (h.compressed_hypertable_id= c.id) 
+        WHERE h.schema_name = 'prom_data' AND h.table_name = metric_table_name;
+
+        --Make the compressed tables freeze on every vacuum run. They won't be changed
+        --later anyway and there is no point requiring another vacuum before wraparound.
+        --Also, make sure autovacuum runs quickly after initial creation, by setting the
+        --threshold low.
+        IF current_setting('server_version_num')::integer >= 130000 THEN
+            EXECUTE FORMAT($$
+                ALTER TABLE %I,%I SET (autovacuum_freeze_min_age=0, autovacuum_freeze_table_age=0, autovacuum_vacuum_insert_threshold=5, autovacuum_vacuum_insert_scale_factor=0.0)
```

Comment:
I don't quite understand the 5 in the
```
autovacuum_vacuum_insert_threshold=5
```
But I imagine the whole table is populated in a single transaction so difference between 5 and 5M is negligible.

Reply:
yeah I changed it to 1.